libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationViewModel.java

```diff
@@ -63,11 +65,11 @@
   private static final String OKHTTP_INSTRUCTION_CACHE = "okhttp-instruction-cache";
   private static final long TEN_MEGABYTE_CACHE_SIZE = 10 * 1024 * 1024;
 
-  public final MutableLiveData<InstructionModel> instructionModel = new MutableLiveData<>();
-  public final MutableLiveData<BannerInstructionModel> bannerInstructionModel = new MutableLiveData<>();
-  public final MutableLiveData<SummaryModel> summaryModel = new MutableLiveData<>();
-  public final MutableLiveData<Boolean> isOffRoute = new MutableLiveData<>();
-  public final MutableLiveData<RouteJunctionModel> routeJunctionModel = new MutableLiveData<>();
+  private final MutableLiveData<InstructionModel> instructionModel = new MutableLiveData<>();
+  private final MutableLiveData<BannerInstructionModel> bannerInstructionModel = new MutableLiveData<>();
+  private final MutableLiveData<SummaryModel> summaryModel = new MutableLiveData<>();
+  private final MutableLiveData<Boolean> isOffRoute = new MutableLiveData<>();
+  private final MutableLiveData<RouteJunctionModel> routeJunctionModel = new MutableLiveData<>();
```

Comment:
For documentation purposes noting that this breaks SemVer cc @langsmith

Reply:
Does this really break anything? These aren't normally external developer facing are they?

Reply:
You're right, although they were `public` and `NavigationViewModel` too (it's not under an `internal` package either) so technically speaking yes, it's SemVer. As discussed we should work on https://github.com/mapbox/mapbox-navigation-android/issues/2864 so we don't need to worry about this anymore.